tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_live_os.py

```diff
@@ -82,9 +82,14 @@ def test_network_required(self):
         """Test the network_required property."""
         assert self.module.network_required is False
 
-    def test_required_space(self):
+    @patch("pyanaconda.modules.payloads.source.live_os.initialization"
+           ".SetUpLiveOSSourceTask.run")
+    def test_required_space(self, task_run):
         """Test the required_space property."""
-        assert self.module.required_space > 0
+        assert self.module.required_space == 0
+
+        self.module._required_space = 12345
+        assert self.module.required_space == 12345
```

Comment:
Please, keep here only the test for zero and add a test similar to `test_handle_setup_task_result` to test other values.